tests/integration/questionnaire/__init__.py

```diff
@@ -12,6 +12,15 @@ def add_person(self, first_name: str, last_name: str):
         number_of_people = self.number_of_people()
         return self.get_list_item_ids()[number_of_people - 1]
 
+    def add_company(self, name: str, number: str, authorised_insurer: str):
+        self.post(
```

Comment:
Since this is only used by `test_questionnaire_list_collector_content.py` and is specific to that test, i think it would be more suitable there